website/static/js/contribAdder.js

```diff
@@ -377,6 +432,42 @@ var AddContributorViewModel = oop.extend(Paginator, {
         var response = JSON.parse(xhr.responseText);
         // Update error message
         this.inviteError(response.message);
+    },
+    /**
+     * get node tree for treebeard from API V1
+     */
+    fetchNodeTree: function() {
+        var self = this;
+        var treebeardUrl = window.contextVars.node.urls.api  + 'tree/';
+        return $.ajax({
+            url: treebeardUrl,
+            type: 'GET',
+            dataType: 'json'
+        }).done(function(response) {
+            self.nodesOriginal = $osf.getNodesOriginal(response[0], self.nodesOriginal);
+            Object.size = function(obj) {
+                var size = 0, key;
+                for (key in obj) {
+                    if (obj.hasOwnProperty(key)) size++;
```

Comment:
Will this always evaluate to true? Instead, could you check the length of `self.nodesOriginal` to determine whether the node has any children?